All Core Devs Meetings/Meeting 58.md

```diff
@@ -0,0 +1,298 @@
+# Ethereum Core Devs Meeting 58 Notes
+### Meeting Date/Time: Friday, March 29, 2019 14:00 UTC
+### Meeting Duration:  1 hours 45 minutes
+### [GitHub Agenda Page](https://github.com/ethereum/pm/issues/89)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=v8Psbo8zY4Y)
+### Moderator: Hudson Jameson
+
+# Summary
+
+### DECISIONS MADE
+
+**DECISION 58.1** The [ProgPoW Carbon Vote](http://www.progpowcarbonvote.com/) signal will be shutting down in 13 days from today. Everyone that wishes to vote will need to do so before this date and ensure they leave their ETH in the address they voted from until after the specified block number.
+
+**DECISION 58.2** Going forward clients can provide an update in the comments in the agenda. If there are any questions or anything specific to discuss a space will be left to do so.
+
+**DECISION 58.3** Going forward Reseach can provide an update in the comments in the agenda. If there are any questions or anything specific to discuss a space will be left to do so.
+
+### ACTIONS REQUIRED
+
+**ACTION 58.1** Cat Herders to look at updating EIP1.
+
+**ACTION 58.2** Review the proposed solutions for Roadmaps in the [Ethereum Magicians](https://ethereum-magicians.org/t/more-frequent-smaller-hardforks-vs-less-frequent-larger-ones/2929) forum to decide if going forward the Core Devs adopts smaller hardforks rather than larger hardforks.
+
+**ACTION 58.3** Vitalik to format the currently proposed EIP-1559 so that it is correctly presented.
+
+**ACTION 58.4** Lane to provide a block number for when the [ProgPoW Carbon Vote](http://www.progpowcarbonvote.com/) will be shutdown.
+
+**ACTION 58.5** Discuss if ProgPoW should continue to be implemented if the Technical Audit is not funded in [Ethereum Magicians](https://ethereum-magicians.org/tags/progpow) or [here](https://ethereum-magicians.org/t/what-has-to-be-done-to-get-progpow-on-ethereum/1361) or [here](https://ethereum-magicians.org/t/motion-to-not-include-progpow-without-audit/3027).
```

Comment:
A lot of conversation has happened here, may be worth linking too: https://ethereum-magicians.org/t/motion-to-not-include-progpow-without-audit/3027

Reply:
Already in the list of links.